### PR TITLE
changes in policies ui

### PR DESCRIPTION
**Pull request by MAS3k**

### CODE REVIEW

File: src/components/Topbar/index.js
Comment (Ghuffran97) [inline src/components/Topbar/index.js:1]: add ? 
 const handleButtonClick = (loggedInUser) => {
    navigate(`${routes.USER_ATTENDANCE_DETAILS}/${loggedInUser?.id}`);
  };

File: src/features/Policy/PolicyContent/index.js
Comment (Ghuffran97) [inline src/features/Policy/PolicyContent/index.js:1]: const isAuthForActions = userRole === ROLES.ADMIN || userRole === ROLES.HR

add ?

File: src/components/Topbar/index.js
Comment (MAS3k) [reply src/components/Topbar/index.js:1]: resolved

File: src/features/Policy/PolicyContent/index.js
Comment (Ghuffran97) [inline src/features/Policy/PolicyContent/index.js:1]: still not resolved. i mentioned the code piece

Comment (MAS3k) [reply src/features/Policy/PolicyContent/index.js:1]: resolved

Comment (MAS3k) [reply src/features/Policy/PolicyContent/index.js:1]: resolved

**Review by Ghuffran97 — APPROVED** [review]

File: src/components/Topbar/index.js
Comment (MAS3k) [reply src/components/Topbar/index.js:1]: resolved

File: src/features/Policy/PolicyContent/index.js
Comment (MAS3k) [reply src/features/Policy/PolicyContent/index.js:1]: resolved

Comment (MAS3k) [reply src/features/Policy/PolicyContent/index.js:1]: resolved

**Pull request merged**